infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -148,7 +148,7 @@ public void handleHeadRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRo
      */
     @ServerHandler(type = CorfuMsgType.WRITE)
     public void write(CorfuPayloadMsg<WriteRequest> msg, ChannelHandlerContext ctx, IServerRouter r) {
-        log.debug("log write: global: {}, streams: {}", msg.getPayload().getToken(),
+        log.trace("log write: global: {}, streams: {}", msg.getPayload().getToken(),
```

Comment:
We need this enabled for debugging.

Reply:
This record actually is pretty useless, it writes a message in the log indicating that the operation has happened but in reality, the entry goes to the batch writer and can stay there (in a queue) for a relatively long time. 
I would prefer using StreamLogFiles to write the message to the log after the fact when actual writing to disk has happened.
Let me double check if I did it in this pr or not

Reply:
> This record actually is pretty useless
False, I found this log very useful for debugging many issues.